src/patternfly/components/Toolbar/toolbar.scss

```diff
@@ -88,19 +88,26 @@
 }
 
 .pf-c-toolbar__filter {
+  display: none;
   flex-basis: 100%;
   order: 1;
   width: auto;
   margin-top: var(--pf-c-toolbar__filter--MarginTop);
   margin-left: var(--pf-c-toolbar__filter--MarginLeft);
+  visibility: hidden;
 
-  @media screen and (max-width: $pf-global--breakpoint--lg) {
-    display: none;
-    visibility: hidden;
+  &.pf-m-expanded {
+    display: flex;
+    visibility: visible;
+  }
+
+  @media screen and (min-width: $pf-global--breakpoint--lg) {
+    display: block;
+    visibility: visible;
 
     &.pf-m-expanded {
-      display: flex;
-      visibility: visible;
+      display: none;
```

Comment:
I think you can just get rid of this block entirely. The filter should be shown on desktop, and you wouldn't want to hide it when you apply `.pf-m-expanded`

Reply:
yep good point, removed